_layouts/default.html

```diff
@@ -6,10 +6,14 @@
     {% include head.html %}
   </head>
 
-  <body class="page-{{ page.title | slugify }} {% if page.layout == 'demo' %}layout-{{ page.layout | slugify }}{% endif %}">
+  <body class="page-{{ page.title | slugify }} layout-{{ page.layout }}">
```

Comment:
Let's re-write this so that the hyphen is only added if `{{ page.layout }}` exists. You should be able to do this with [prepend](https://shopify.github.io/liquid/filters/prepend/). We can then potentially use `layout` as a more generic layout class, if the need arises.